packages/generator/src/helper.ts

```diff
@@ -1,68 +1,76 @@
 import * as fontkit from 'fontkit';
-import * as pdfLib from '@pdfme/pdf-lib';
-import type { GeneratorOptions, Template, PDFRenderProps, Plugin } from '@pdfme/common';
-import type { Schema, Plugins } from '@pdfme/common';
+import type {
+  Schema,
+  Plugins,
+  GeneratorOptions,
+  Template,
+  PDFRenderProps,
+  Plugin,
+} from '@pdfme/common';
 import { builtInPlugins } from '@pdfme/schemas';
 import { PDFPage, PDFDocument, PDFEmbeddedPage, TransformationMatrix } from '@pdfme/pdf-lib';
-import { getB64BasePdf, BasePdf } from '@pdfme/common';
+import { getB64BasePdf, BasePdf, isBlankPdf, mm2pt } from '@pdfme/common';
```

Comment:
this can be combined with the import above

Reply:
Oh, you are right